AnnoyancesFilter/sections/remains.txt

```diff
@@ -73,7 +69,6 @@ bursa.com##.bg--gri[style="padding: 10px 0;"]
 m.pornxs.com###im_scroll_close_button
 m.smutty.com##.ui-content .center[style*="margin-"]:not([id])
 m.empflix.com##.bannerBlock
-t-online.de##div[data-dy^="Partner:"]:not([data-dy^="Partner:Telekom"])
```

Comment:
Rule is found on t-online.de when I check the site, so please add it to German filter, thank you.

Reply:
I do have the "Allow edits by maintainers" button turned on, you know. But sure thing, I'll take care of it. Give me 2min.

Reply:
Oh, wait, now I remember. `t-online.de##div[data-dy*="Partner:"]` is already in both AdGuard German and in EasyList Germany, thus making the entry superfluous after all.